tests/diffpy/utils/scattering_objects/test_diffraction_objects.py

```diff
@@ -231,6 +231,158 @@ def test_diffraction_objects_equality(inputs1, inputs2, expected):
     assert (diffraction_object1 == diffraction_object2) == expected
 
 
+def _test_valid_diffraction_objects(actual_diffraction_object, function, expected_array):
+    """Checks the behavior of the DiffractionObject:
```

Comment:
Test functions don't need docstring, though if it's ok to leave this now you did it.